data-model/digitalobjects/0.2.0/digital-specimens/schema/identifications.json

```diff
@@ -0,0 +1,365 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.2.0/digital-specimens/identifications.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.2.0",
+  "type": "object",
+  "properties": {
+    "dwc:identificationID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationID",
+      "examples": [
+        "9992"
+      ]
+    },
+    "???:identificationType": {
+      "type": "string",
+      "description": "Unclear yet",
+      "$comment": "Unknown what this field should be"
+    },
+    "???:taxonFormula": {
```

Comment:
this can be captured in identificationRemarks. I think this probably came from https://dwc.tdwg.org/list/#dwc_verbatimIdentification. "This term is meant to allow the capture of an unaltered original identification/determination, including identification qualifiers, hybrid formulas, uncertainties, etc. This term is meant to be used in addition to dwc:scientificName (and dwc:identificationQualifier etc.), not instead of it."

Reply:
No taxonFormula comes from Arctos and is a way to combine multiple valid taxonomic identification. For example a fur hat with multiple species used in it can get a taxonFormula. The verbatimIdentification is just a transcription of the label. Full explanation is in one of the GBIF UM videos